### PR TITLE
Saga.ReplyToOriginator sets the wrong MessageIntent

### PR DESCRIPTION
**Pull request by johnsimons**
When a reply is done the `MessageIntent` for  should be set to `Reply`.

`Saga.ReplyToOriginator` is incorrectly assigning it to `Send`.

### CODE REVIEW

**Comment by johnsimons** [comment]
@Particular/nservicebus this one is ready to be reviewed.

I had to put a ["hack"](https://github.com/Particular/NServiceBus/pull/1974/files#diff-6cedb33cdd3ef270fa45530546a4a899R26) for this to work, it feel a bit weird we need to hack this, thoughts ?

File: src/NServiceBus.Core/Saga/Saga.cs [hunk 1]
```diff
@@ -157,7 +157,20 @@ protected virtual void ReplyToOriginator(object message)
                 throw new Exception("Entity.Originator cannot be null. Perhaps the sender is a SendOnly endpoint.");
             }
 
-            Bus.Send(message, new SendOptions(Entity.Originator, Entity.OriginalMessageId));
+            var sendOptions = new SendOptions(correlationId: Entity.OriginalMessageId);
+            sendOptions.AsReplyTo(Entity.Originator);
+            sendOptions.ExtensionContext.Set(new WrapperForReplyToOriginator(true));
+            Bus.Send(message, sendOptions);
```
Comment (andreasohlund) [inline src/NServiceBus.Core/Saga/Saga.cs]: Should we add a way to set the message intent explicitly here instead? (still a hack but less so IMO)

Comment (andreasohlund) [inline src/NServiceBus.Core/Saga/Saga.cs]: Or a bus.Reply where we can set the correlation id and address explicitly?

**Comment by andreasohlund** [comment]
@johnsimons I think I have this one solved in:

https://github.com/Particular/NServiceBus/pull/2737/files#diff-9ba7ec6db6dafc79acd3a5fcbe4d2b6eR165

I'll port your tests over and close this one as soon as #2737 is complete.

Does this make sense?

**Comment by yvesgoeleven** [comment]
Looks like this issue is present in 5.2 as well

**Comment by andreasohlund** [comment]
Ported tests over to #2737 https://github.com/Particular/NServiceBus/commit/5013b5dba16973944e3e4dac2d5d355578395dbe



@yvesgoeleven is fixing v5 separately

Closing this one

**Pull request closed without merging**